sdk-extensions/autoconfigure/README.md

```diff
@@ -35,6 +35,15 @@ and not something we can control. If you require logging during shutdown hooks,
 that might shut itself down in a shutdown hook, thus suppressing your log messages. See this [JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253)
 for more details.
 
+It is possible to skip the automatic registration of the shutdown hook by using the following configuration item:
+
+| System property                  | Environment variable             | Description                                                          |
+|----------------------------------|----------------------------------|----------------------------------------------------------------------|
+| otel.java.register.shutdown.hook | OTEL_JAVA_REGISTER_SHUTDOWN_HOOK | If `false` skips the shutdown hook registration. Default is `true`. |
```

Comment:
If we're going to introduce non-officially-specified env vars, I think we should keep them in the `experimental` namespace, so if the spec does end up supporting this idea that we won't have to support two non-experimental options for the same thing.